### DISCUSSION TITLE
Localhost not showing Quarto/Rstudio pdf file

### DISCUSSION THREAD

**Discussion by dkStevensNZed** [quarto-dev/quarto-cli#3721] [Q&A] [answered]
I've been successfully using Quarto/RStudio for ~6 months, although the "rendering to pdf file" behavior has been erratic. After installing latest RStudio and Quarto with the latest R version, the .pdf file is produced and a localhost window in my browser (latest Firefox) opens, but a message 'Not found' on a blank white screen in the browser window is all I get. I can read it in Acrobat but not in the browser. Prior to installing all new versions, the .pdf file would show up in the browser about 1/2 the time. What should I look for to fix this up?

**Comment by mcanouil** [comment]
Could you provide your operating system version, RStudio IDE version and Quarto version if not the one embedded in RStudio IDE?

**Comment by mlozanoqf** [comment, marked as answer]
Try:
```
project:
  output-dir: out
```

**Comment by dkStevensNZed** [comment]
Thank you so much. That did the trick!
